services/ui-src/src/components/tables/EntityStatusIcon.tsx

```diff
@@ -5,36 +5,56 @@ import unfinishedIcon from "assets/icons/icon_error_circle_bright.png";
 import unfinishedIconDark from "assets/icons/icon_error_circle.png";
 import successIcon from "assets/icons/icon_check_circle.png";
 import successIconDark from "assets/icons/icon_check_circle_dark.png";
+import closedIcon from "assets/icons/icon_circle-minus-gray.png";
+
+export type EntityStatusType = string | boolean | undefined;
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Nit: I think we can remove this eslint now

Reply:
same as the other one, going to put it in a clean up ticket